content/docs/add-react-to-a-website.md

```diff
@@ -1,202 +1,202 @@
 
-[View the full example source code](https://gist.github.com/gaearon/faa67b76a6c47adbab04f739cba7ceda)
+[Örnek kaynak kodunun tamamını görüntüleyin](https://gist.github.com/gaearon/faa67b76a6c47adbab04f739cba7ceda)
 
-[Download the full example (2KB zipped)](https://gist.github.com/gaearon/faa67b76a6c47adbab04f739cba7ceda/archive/9d0dd0ee941fea05fd1357502e5aa348abb84c12.zip)
+[Tüm örneği indirin (2KB sıkıştırılmış)](https://gist.github.com/gaearon/faa67b76a6c47adbab04f739cba7ceda/archive/9d0dd0ee941fea05fd1357502e5aa348abb84c12.zip)
 
->Note
+>Not
 >
->This strategy is mostly useful while React-powered parts of the page are isolated from each other. Inside React code, it's easier to use [component composition](/docs/components-and-props.html#composing-components) instead.
+>Bu strateji, sayfanın React destekli bölümleri birbirinden izole edilirken çoğunlukla kullanışlıdır. Bunun yerine React kodunun içinde [bileşen derlemesini](/docs/components-and-props.html#composing-components) kullanmak daha kolaydır.
 
-### Tip: Minify JavaScript for Production {#tip-minify-javascript-for-production}
+### İpucu: Üretim İçin JavaScript'i küçültün {#tip-minify-javascript-for-production}
 
-Before deploying your website to production, be mindful that unminifed JavaScript can significantly slow down the page for your users.
+Web sitenizi yayına sokmadan önce, küçültülmemiş JavaScript'in sayfanızı kullanıcılarınız için önemli ölçüde yavaşlatabileceğine dikkat edin.
 
-If you already minify the application scripts, **your site will be production-ready** if you ensure that the deployed HTML loads the versions of React ending in `production.min.js`:
+Uygulama komut dosyalarını küçültürseniz ve yayınlamaya hazır hale getirilen HTML'in de React'ın `production.min.js` içinde biten sürümlerini yüklediğinden emin olursanız **siteniz üretime hazır** olur:
 
 ```js
 <script src="https://unpkg.com/react@16/umd/react.production.min.js" crossorigin></script>
 <script src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js" crossorigin></script>
 ```
 
-If you don't have a minification step for your scripts, [here's one way to set it up](https://gist.github.com/gaearon/42a2ffa41b8319948f9be4076286e1f3).
+Komut dosyalarınız için bir küçültme adımınız yoksa, [ayarlamanın bir yolu budur](https://gist.github.com/gaearon/42a2ffa41b8319948f9be4076286e1f3).
 
-## Optional: Try React with JSX {#optional-try-react-with-jsx}
+## İsteğe bağlı: React'ı JSX ile deneyin {#optional-try-react-with-jsx}
 
-In the examples above, we only relied on features that are natively supported by the browsers. This is why we used a JavaScript function call to tell React what to display:
+Yukarıdaki örneklerde, yalnızca tarayıcılar tarafından doğal olarak desteklenen özelliklere itibar ettik. Bu yüzden React’a ne göstereceğini söylemek için bir JavaScript fonksiyon çağrısı kullandık:
```

Comment:
React kelimesi yabancı bir kelime olduğu için ayrılan ekler de okunduğu gibi ayrılmalıdır. "React'e" daha doğru bir kullanımdır.

Örneğin:
Yanlış: Twitter'e girdin mi?
Doğru: Twitter'a girdin mi?